package.json

```diff
@@ -57,5 +58,76 @@
     "src/**/*.(js|jsx)": [
       "eslint src/**/*.{js|jsx}"
     ]
+  },
+  "stylelint":{
+    "rules":{
+      "at-rule-name-case": "lower",
```

Comment:
As I noticed we have missing `indentation` rule and thanks to that we may have different indents without error thrown. I would recommend to specify any. https://stylelint.io/user-guide/rules/indentation

Reply:
I think `indentInsideParens: "once-at-root-twice-in-block"` will do. Am I right?